packages/meditrak-server/src/routes/importSurveys/Validator/JsonFieldValidator.js

```diff
@@ -20,6 +18,12 @@ export class JsonFieldValidator extends BaseValidator {
     return true;
   }
 
+  getConfig(rowIndex) {
+    const fieldName = this.getFieldName();
+    const field = this.getField(rowIndex, fieldName);
+    return this.parseField(field);
+  }
+
```

Comment:
I added this function (so that ConfigValidator can have access to the whole config in case a property is dependent on another property and want to validate something between that) but ended up not using it. However I still keep it cause I think it can be useful later on